src/main/java/net/boomerangplatform/service/crud/TeamServiceImpl.java

```diff
@@ -182,7 +179,7 @@ public void createFlowTeam(String higherLevelGroupId, String teamName) {
     final List<TeamWorkflowSummary> teamWorkFlowSummary = new LinkedList<>();
     
     List<FlowTeamEntity> flowTeam = null;
-    if (flowExternalUrlTeam.isBlank()) {
+    if (!flowExternalUrlTeam.isBlank()) {
       flowTeam = this.cicdService.getCICDTeams();
```

Comment:
We will need to come back to this and use the URL provided? it may not be CICD...